core/tee/tee_rpmb_fs.c

```diff
@@ -26,21 +26,24 @@
  */
 
 #include <assert.h>
-#include <kernel/tee_common.h>
 #include <kernel/mutex.h>
 #include <kernel/panic.h>
+#include <kernel/tee_common.h>
 #include <kernel/tee_common_otp.h>
+#include <kernel/tee_misc.h>
 #include <kernel/thread.h>
 #include <mm/core_memprot.h>
 #include <mm/tee_mm.h>
 #include <optee_msg_supplicant.h>
 #include <stdlib.h>
-#include <string.h>
 #include <string_ext.h>
+#include <string.h>
```

Comment:
I'm sorting with `sort`.
Actually I'm selecting the lines in vim and and then pipe it to sort with `!sort`.